### ISSUE TITLE
Customizable prefixes for upgrade commits and pin dependencies commits

### ISSUE THREAD

**Issue by NearAutomata** [renovatebot/renovate#2351]
**What would you like Renovate to be able to do?**
Recently I've adapted the use of emojis for commit prefixes. Hence I'd like to use :arrow_up: as a "generic" commit prefix for renovate upgrade commits but also :pushpin: for those that pin dependencies in particular therefore I'd like to have configuration options for both cases with custom prefixes.

**Describe the solution you'd like**
Seing a similar idea that was proposed in https://github.com/renovatebot/renovate/issues/1867 I'd like to see a `pinDependenciesCommitMessagePrefix` (definitely need a better name) in addition to the `commitMessagePrefix` which would allow me to define both of my prefixes. 

**Describe alternatives you've considered**
An alternative may be an option to enable "standard" emojis described [here](https://github.com/carloscuesta/gitmoji/) as prefixes. This approach would not allow customization though.

**Comment by rarkins** [comment]
Hi, this should already be possible today via regular configuration using a combination of [semanticCommitType](https://renovatebot.com/docs/configuration-options/#semanticcommittype) and [pin](https://renovatebot.com/docs/configuration-options/#pin). If any questions on that configuration please raise an issue in the [config-help](https://github.com/renovatebot/config-help) repo describing what you've tried and what failed, etc.

A PR would also be welcome here if you think any documents would be improved by describing how to do this for others. Thanks

**Comment by NearAutomata** [comment]
Thank you for pointing me to these configuration options. It seems I haven't understood what they were useful for in the first place.

**Comment by NearAutomata** [comment]
Ultimately I ended up without using `semanticCommitType` and only used `pin` to rewrite my default prefix. I skipped onboarding with my custom `renovate.json` already in the repo:
```json
{
    "extends": [
        "config:base"
    ],
    "commitMessagePrefix": ":arrow_up:",
    "pin": {
        "commitMessagePrefix": ":pushpin:"
    },
    "rangeStrategy": "pin"
}
```
Of course this setup assumes that renovate doesn't do any other types of commits, otherwise ⬆️ wouldn't make too much sense. Anyways, if someone stumbles upon this use case, just refer to my config.
